bottlecap/src/main.rs

```diff
@@ -84,6 +95,12 @@ fn register() -> Result<RegisterResponse> {
 }
 
 fn main() -> Result<()> {
+    tracing_subscriber::fmt()
+        .json()
+        .with_env_filter(EnvFilter::from_default_env())
```

Comment:
We should set the Env filter based on what we read from config. For instance, this could be set in the datadog.yaml